### ISSUE TITLE
Error when drawing circuits generated by StandardRB

### ISSUE THREAD

**Issue by eendebakpt** [Qiskit/qiskit-terra#9145]
### Informations

- **Qiskit Experiments version**: 0.4.0
- **Python version**: 3.10
- **Operating system**: Windows

### What is the current behavior?

Running this script:

```
import qiskit
from qiskit_experiments.library import StandardRB
import qiskit_experiments
import numpy as np
print(qiskit.version.QiskitVersion())
print(qiskit_experiments.version.get_version_info())


lengths = np.arange(1, 600, 200)
exp1 = StandardRB([0], lengths, num_samples=4, seed=1010)
circuits=exp1.circuits()
circuits[0].draw()
```
results in an exception
```
{'qiskit-terra': '0.22.2', 'qiskit-aer': '0.11.1', 'qiskit-ignis': None, 'qiskit-ibmq-provider': '0.19.2', 'qiskit': None, 'qiskit-nature': None, 'qiskit-finance': None, 'qiskit-optimization': None, 'qiskit-machine-learning': None}
0.4.0

  ...

  File "C:\develop\env310\lib\site-packages\qiskit\visualization\circuit\_utils.py", line 556, in slide_from_left
    if node.op.condition:

AttributeError: 'Clifford' object has no attribute 'condition'
```
### Steps to reproduce the problem

See the minimal example above

### What is the expected behavior?

The circuit should be plotted

**Comment by 1ucian0** [comment]
This is related to #9159 as `Clifford` operations are not instructions.

**Comment by 1ucian0** [comment]
Fixed by #8907
